pennylane/resource/first_quantization.py

```diff
@@ -251,61 +310,119 @@ def norm(n, eta, omega, error, br=7, charge=0):
             raise ValueError("system charge must be an integer.")
 
         l_z = eta + charge
-
+        alpha = 0.0248759298  # same as the pseudopotential/AE code
         # target error in the qubitization of U+V which we set to be 0.01 of the algorithm error
-        error_uv = 0.01 * error
+        error_uv = alpha * error
 
         # taken from Eq. (22) of PRX Quantum 2, 040332 (2021)
         n_p = int(np.ceil(np.log2(n ** (1 / 3) + 1)))
 
         n0 = n ** (1 / 3)
-        lambda_nu = (  # expression is taken from Eq. (F6) of PRX 8, 011044 (2018)
-            4 * np.pi * (np.sqrt(3) * n ** (1 / 3) / 2 - 1)
-            + 3
-            - 3 / n ** (1 / 3)
-            + 3 * integrate.nquad(lambda x, y: 1 / (x**2 + y**2), [[1, n0], [1, n0]])[0]
-        )
+
+        bmin = 2 * np.pi / omega ** (1 / 3) if cubic else bmin
         n_m = int(
-            np.log2(  # taken from Eq. (132) of PRX Quantum 2, 040332 (2021)
-                (2 * eta)
-                / (error_uv * np.pi * omega ** (1 / 3))
-                * (eta - 1 + 2 * l_z)
-                * (7 * 2 ** (n_p + 1) - 9 * n_p - 11 - 3 * 2 ** (-1 * n_p))
+            np.ceil(
+                np.log2(  # taken from Eq. (132) of PRX Quantum 2, 040332 (2021)
```

Comment:
It is definitely the right reference if we are talking about cubic lattices. 
For non cubic lattices, this is an equation derived from [our work](https://arxiv.org/abs/2302.07981) , but it doesn't get its own equation in the paper, so I can't give you a ref.
However, since the change from cubic to noncubic is rather easy to derive in this case, I believe it's best to leave the reference as it is, just maybe change the comment to `we can easily generalize the cubic lattice formula in Eq. (132) of PRX Quantum 2, 040332 (2021)` to this equation, or sth along these lines.